manual/root_macros_and_shared_libraries/index.md

```diff
@@ -199,385 +180,35 @@ The `$ROOTSYS/include` directory is automatically appended to the include path.
 To get the include path, type:
 
 ```
-   root[] .include
+root [0] .include
 ```
 
 To append the include path, type:
 
 ```
-   root[] .include $HOME/mypackage/include
+root [0] .include $HOME/mypackage/include
 ```
 
-Append the following line in the ROOT macro to include the include path:
+Add the following line in the ROOT macro to append a new path to the existing include paths:
 
 ```
-   gSystem->AddIncludePath(" -I$HOME/mypackage/include")
+gSystem->AddIncludePath(" -I$HOME/mypackage/include")
 ```
 
 To overwrite an existing include path, type:
 
 ```
-   gSystem->SetIncludePath(" -I$HOME/mypackage/include")
+gSystem->SetIncludePath(" -I$HOME/mypackage/include")
 ```
 
 To add a static library that should be used during linking, type:
```

Comment:
I think the example below is for a shared library (not static)

Reply:
Okay I'll change it